core/templates/components/state-editor/state-editor-properties-services/state-editor.service.spec.ts

```diff
@@ -145,5 +231,323 @@ describe('Editor state service', () => {
       val: 'ca_choices_1',
       label: 'Choice 2',
     }]);
+    expect(
+      ecs.getAnswerChoices(
+        'NotDragAndDropSortInput',
+        customizationArgsForItemSelectionAndDragAndDropInput)
+    ).toEqual(null);
+  });
+
+  it('should return null when getting answer choices' +
+    ' if interactionID is empty', () => {
+    expect(ecs.getAnswerChoices('', {
+      choices: {
+        value: [
+          new SubtitledHtml('Choice 1', ''),
+          new SubtitledHtml('Choice 2', '')
+        ]
+      }
+    })).toBe(null);
+  });
+
+  it('should return if exploration is whitelisted or not', () => {
+    expect(ecs.isExplorationWhitelisted()).toBe(false);
+    ecs.explorationIsWhitelisted = true;
+    expect(ecs.isExplorationWhitelisted()).toBe(true);
+    ecs.explorationIsWhitelisted = false;
+    expect(ecs.isExplorationWhitelisted()).toBe(false);
+  });
+
+  it('should return if correctness feedback is enabled or not', () => {
+    expect(ecs.correctnessFeedbackEnabled).toBe(null);
+    ecs.correctnessFeedbackEnabled = true;
+    expect(ecs.correctnessFeedbackEnabled).toBe(true);
+    ecs.correctnessFeedbackEnabled = false;
+    expect(ecs.correctnessFeedbackEnabled).toBe(false);
+  });
+
+  it('should update state content editor initialised to true', () => {
```

Comment:
`should initialise state content editor`. Use the same format below.

Reply:
Done